docs/cloud/automation/README.md

```diff
@@ -164,9 +164,9 @@ Before provisioning a database cluster, you need a network, which the cluster wi
 |:--------------------|:---------|:---------------------------------------------------------------------------|
 | `name`              | `string` | *Required*, the new network name.                                          |
 | `project_id`        | `string` | *Required*, the project ID of the new network (see [Projects](#projects))  |
-| `resource_provider` | `string` | *Required*, the network cloud provider (`aws` , `gcp` , `azure`).          |
+| `resource_provider` | `string` | *Required*, the network cloud provider (`aws`, `gcp`, `azure`).            |
```

Comment:
Trailing space

Reply:
it's actually not trailing whitespace - I moved last '|' symbol. You can see it in a source code

<img width="901" alt="Screenshot 2022-04-01 at 01 31 30" src="https://user-images.githubusercontent.com/1574361/161166059-361af341-6e0f-4188-ab1a-12d47b20f9d1.png">